x-pack/filebeat/module/activemq/log/_meta/fields.yml

```diff
@@ -0,0 +1,21 @@
+- name: log
+  type: group
+  description: >
+    ActiveMQ application log lines.
+  fields:
+    - name: level
```

Comment:
If I am not mistaken you can omit this, since it will be mapped automatically from ECS. Need to verify though.

Reply:
Thanks! Changed, seems to be working. Let CI builder verify it.